### PR TITLE
Nikita Proskurnikov, HW1

### PR DESCRIPTION
**Pull request by Iandmee**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+    elem.title = '';
+    elem.setAttribute("tooltip", title);
+    const tooltipWrap = document.createElement("div");
+    tooltipWrap.className = 'tooltip';
+    tooltipWrap.appendChild(document.createTextNode(title));
+    const padding = 35;
+    const linkProps = elem.getBoundingClientRect();
+    const tooltipProps = tooltipWrap.getBoundingClientRect();
+    const topPos = linkProps.top - (tooltipProps.height + padding);
+    tooltipWrap.setAttribute('style','top:'+topPos+'px;'+'left:'+linkProps.left+'px;')
```
Comment (coolassassin) [inline main.js]: Style is an object of an element
You can replace it like that:
```javascript
tooltipWrap.style.top = topPos + 'px';
tooltipWrap.style.left = linkProps.left + 'px';
```

Or like that:
```javascript
tooltipWrap.style.top = `${topPos}px`;
tooltipWrap.style.left = `${linkProps.left}px`;
```

File: main.js [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+    elem.parentNode.insertBefore(tooltipWrap, elem);
+}
+function removeTooltip(elem){
+    elem.title = elem.getAttribute("tooltip");
+    elem.removeAttribute("tooltip");
+    document.querySelector(".tooltip").remove();
```
Comment (coolassassin) [inline main.js]: You will remove first random tooltip like so. What if you have 10 buttons with tooltips on click?

File: main.js (hunk 1 above)
Comment (Iandmee) [reply main.js]: Thanks, I have fixed all

File: main.js (hunk 2 above)
Comment (Iandmee) [reply main.js]: I replaced it by `queryAll`

**Pull request closed without merging**